meta.json

```diff
@@ -1,4 +1,4 @@
 {
-    "release": "1",
+    "release": "2",
```

Comment:
These changes indeed affect only the metadata of `tezos-baking` package (since it only changes the `.services` files). However, we did change the sources of `tezos-baking` package as well (by modifying `tezos_setup_wizard.py`), so I think we should bump `letter_version` and leave the `release` as it is